Fabric.Authorization.API/Modules/UsersMetadataModule.cs

```diff
@@ -22,7 +22,7 @@ public class UsersMetadataModule : BaseMetadataModule
         private readonly Parameter _userIdParameter = new Parameter
         {
             Name = "userId",

```

Comment:
Do we also need to change the name of the parameter from userId to subjectId?

Reply:
That is changed in another pull request where I have the swagger doc updates.